cmd/kes/log.go

```diff
@@ -65,8 +74,13 @@ func logTrace(args []string) error {
 		fmt.Fprintf(cli.Output(), logTraceCmdUsage, cli.Name())
 	}
 
-	var jsonOutput bool
-	var insecureSkipVerify bool
+	var (
+		typeFlag   string
+		jsonOutput bool
+
+		insecureSkipVerify bool
+	)
+	cli.StringVar(&typeFlag, "type", "audit", "Log event type [ audit | error ]")
```

Comment:
is there a good reason for you to use Go flags @aead  and not `minio/cli` ?

Reply:
The argument works the other way around.
Currently, the Go standard library works for this project. So, there is no need (yet) to add a 3rd party library for this task.